src/main/webapp/src/app/create-gardens-form/create-gardens.component.html

```diff
@@ -0,0 +1,32 @@
+<mat-card class="mat-elevation-z8 form">
+  <mat-card-title>
+    Create Garden Form
+  </mat-card-title>
+
+  <mat-card-content>
+    <mat-form-field>
+      <input matInput placeholder="Name of Garden" /> </mat-form-field
+    ><br />
+
+    <mat-form-field>
+      <input matInput placeholder="Name of Admin" /> </mat-form-field
+    ><br />
+
+    <mat-form-field>
+      <input matInput placeholder="Address of Garden" /> </mat-form-field
```

Comment:
nit: pretty funky looking whitespace here as well. is this from `make format`?

Reply:
Yup both are from 'make format'

Reply:
Yeah, a lot of these formatter/linter settings need to be tweaked -- look at Jake's comments on #1 for more examples.